### PR TITLE
python3Packages.videocr: init at 0.1.6

### PR DESCRIPTION
**Pull request by ozkutuk**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/videocr/default.nix [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    fuzzywuzzy
+  ];
+
+  postPatch = ''
+    substituteInPlace setup.py \
+      --replace "opencv-python" "opencv"
+    substituteInPlace videocr/constants.py \
+      --replace "master" "main"
+    substituteInPlace videocr/video.py \
+      --replace '--tessdata-dir "{}"' '--tessdata-dir="{}"'
```
Comment (mweinelt) [inline pkgs/development/python-modules/videocr/default.nix:32]: Can we upstream those changes?

File: pkgs/development/python-modules/videocr/default.nix [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  '';
```
Comment (mweinelt) [inline pkgs/development/python-modules/videocr/default.nix:33]:
```suggestion
  '';
  
  # project has no tests
  doCheck = false;
```

File: pkgs/development/python-modules/videocr/default.nix (hunk 1 above)
Comment (ozkutuk) [reply pkgs/development/python-modules/videocr/default.nix:32]: There is are multiple issues on upstream regarding the primary branch renaming (https://github.com/apm1467/videocr/issues/36, https://github.com/apm1467/videocr/issues/39). Considering there have been no responses from the author on any of the issues and the last commit to the repo is in 2019, I think it is safe to say that the project is abandonware.

**Comment by mweinelt** [comment]
Can you squash all commits related to the package into one, so we only have the maintainer and the package commits left?

**Pull request merged**